src/styles/variables.scss

```diff
@@ -1,10 +1,16 @@
+@import "fonts";
+
 $primary-color: #0F95CE;
 $secondary-color: #002E5D;
-$terciary-color: #F2FBFF;
+$tertiary-color: #F2FBFF;
```

Comment:
💯 love this styling names and great spirit on addressing typos, they tend to be there forever XD